docs/reST/ref/display.rst

```diff
@@ -148,26 +148,18 @@ required).
 
    The flags argument controls which type of display you want. There are
    several to choose from, and you can even combine multiple types using the
-   bitwise or operator, (the pipe "|" character). If you pass ``0`` or no flags
-   argument it will default to a software driven window. Here are the display
+   bitwise or operator, (the pipe "|" character). Here are the display
    flags you will want to choose from:
 
    ::
 
       pygame.FULLSCREEN    create a fullscreen display
-      pygame.DOUBLEBUF     recommended for HWSURFACE or OPENGL
-      pygame.HWSURFACE     hardware accelerated, only in FULLSCREEN
+      pygame.DOUBLEBUF     (obsolete in pygame 2) recommended for HWSURFACE or OPENGL
+      pygame.HWSURFACE     (obsolete in pygame 2) hardware accelerated, only in FULLSCREEN
       pygame.OPENGL        create an OpenGL-renderable display
       pygame.RESIZABLE     display window should be sizeable
       pygame.NOFRAME       display window will have no border or controls
-
-
-   Pygame 2 has the following additional flags available.
```

Comment:
I think we still need to say which flags are new in pygame 2. (We're still at a point for some people where they want their games to run on pygame 2 and 1, and pygame 1 is still more widely installable)

Reply:
Yes, definitely! I knew I considered that but I had forgotten how. There was pre existing ..versionchanged that enumerates which ones are new in pygame 2, I just though the two tables would look much prettier as one 😄